hal/inc/interrupts_hal.h

```diff
@@ -91,9 +91,9 @@ extern "C" {
 
 void HAL_Interrupts_Init(void);
 void HAL_Interrupts_Uninit(void);
-void HAL_Interrupts_Attach(uint16_t pin, HAL_InterruptHandler handler, void* data, InterruptMode mode, HAL_InterruptExtraConfiguration* config);
-void HAL_Interrupts_Detach(uint16_t pin);
-void HAL_Interrupts_Detach_Ext(uint16_t pin, uint8_t keepHandler, void* reserved);
+int  HAL_Interrupts_Attach(uint16_t pin, HAL_InterruptHandler handler, void* data, InterruptMode mode, HAL_InterruptExtraConfiguration* config);
```

Comment:
Nitpicking: no need for two spaces :)

Reply:
Ok, two spaces just for column alignment. I will follow your suggetion.